Packs/KELARaDark/Integrations/RaDark/RaDark.py

```diff
@@ -0,0 +1,610 @@
+    try:
+        incident_id = str(args.get('incident_id', ''))
+        incident_data = client.incident_get_items(incident_id=incident_id)
+        if isinstance(incident_data, dict) and isinstance(incident_data.get('data'), dict) and \
+                isinstance(incident_data['data'].get('incidents'), list) and len(incident_data['data']['incidents']) > 0:
+            parsed_data, readable_output = parse_incident_markdown_table(incident_data['data'], incident_id)
+        else:
+            readable_output = f'No data found for item ID: {incident_id}'
+            parsed_data = []
+        return CommandResults(
+            readable_output=readable_output,
+            outputs_prefix='Radark.itemDetails',
+            outputs_key_field='items',
+            outputs={'items': parsed_data, "details": readable_output})
+    except Exception as e:
+        demisto.error(traceback.format_exc())  # print the traceback
+        return_error(f'Failed to execute command.\nError:\n {str(e)} {incident_data}')
+        return None
+
+
+def email_enrich_command(client: Client, args: Dict[str, Any]) -> Union[CommandResults, None]:
+    try:
+        email = str(args.get('email', ''))
+        email_data = client.email_enrich_data(email=email)
+        if isinstance(email_data, dict) and isinstance(email_data.get('data'), dict) and \
+                isinstance(email_data['data'].get('incidents'), list) and len(email_data['data']['incidents']) > 0:
+            parsed_data, readable_output = parse_email_enrichment_markdown_table(email_data['data'])
+        else:
+            readable_output = f'No data found for email: {email}'
+            parsed_data = None
+        return CommandResults(
+            readable_output=readable_output,
+            outputs_prefix='Radark.emailDetails',
+            outputs_key_field='emails',
+            outputs={'emails': parsed_data})
+    except Exception as e:
+        demisto.error(traceback.format_exc())  # print the traceback
+        return_error(f'Failed to execute command.\nError:\n {str(e)} {email}')
+        return None
+
+
+def item_handle_command(client: Client, args: Dict[str, Any]) -> Union[CommandResults, None]:
+    try:
+        item_id = str(args.get('item_id', ''))
+        action_res = client.action_on_item(item_id=item_id, action="handled")
+        if isinstance(action_res, dict) and isinstance(action_res.get('data'), dict) \
+                and action_res['data'].get('value'):
+            readable_output = f'Item ID ({item_id}) marked as handled'
+            return CommandResults(readable_output=readable_output)
+        else:
+            raise Exception("RaDark Error: Mark item action failed!")
+    except Exception as e:
+        demisto.error(traceback.format_exc())  # print the traceback
+        return_error(f'Failed to execute command on {item_id}.\nError:\n {str(e)}')
+        return None
```

Comment:
It is unnecessary as the exception will be caught in the main function.
```suggestion
        raise Exception(f'RaDark Error: Failed to execute command on {item_id}.\nError:\n {str(e)}')
```